src/app/screens/AddMember/i18n.js

```diff
@@ -1,10 +1,20 @@
 import i18next from 'i18next';
 
-i18next.addResources('es', 'AddMemberToOrganization', {
-  title: 'Agregar miembro a la organización de Wolox',
-  addButton: 'Agregar',
-  labelMenu: 'Agregar miembro a org',
-  userInput: 'Usuario de Github',
-  successMessage: 'El miembro se agregó con éxito.',
-  failedMessage: 'Hubo un error al agregar el miembro a la organización.'
-});
+i18next.addResourceBundle(
```

Comment:
Why did you change the i18n format?

Reply:
The formatter did it. I only changed the addResources for addResourceBundle method. I needed to modify the structure to add nested resources.